joomla/joomla-cms
# Adding an alternate layout parameter to custom fields

Pull Request for Issue #18162 .

### Summary of Changes
Adds a new parameter "Layout" to custom fields which allows to select a different field layout for the automatic display.

### Testing Instructions
* Create alternative layouts for a custom field. The original one is `components/com_fields/layout/field/render.php`, it can be overriden at component and template level, so for com_content in `components/com_content/layout/field/`, `templates/your_template/html/layouts/com_content/field` and `templates/your_template/html/layouts/com_fields/field`. The file isn't allowed to contain an underscore.
* Create a custom field and change the layout for that field, assign some value to it
* Check in frontend if that layout is used as expected.


### Expected result
Layout is used as specified


### Actual result
Not possible to choose a different layout


### Documentation Changes Required
Since it is a new feature, it has to be documented.
